CenterForOpenScience/osf-pigeon
# [ENG-1997] Modify Pigeon to use IA library for uploads

## Purpose

We should use the IA library for uploading files onto the internet archive rather than the low level boto

## Changes

Rewrote upload function to utilize IA library's upload, including ability to upload entire directories.

## QA Notes

  - What is the level of risk?
Minimal
    - Is this an additive or subtractive change, other? Modification
  - How can QA verify? (Through UI, API, AdminApp or AdminAdminApp?) Check all files are properly uploaded to IA
-->

## Documentation

N/A

## Side Effects

Shouldn't be.

## Ticket

https://openscience.atlassian.net/browse/ENG-1997
